editor/src/clj/editor/scene.clj

```diff
@@ -1215,7 +1219,19 @@
               (ui/user-data! image-view ::last-renderables renderables)
               (ui/user-data! image-view ::last-frame-version frame-version)
               (scene-cache/prune-context! gl)
-              (reset! async-copy-state-atom (scene-async/finish-image! (scene-async/begin-read! @async-copy-state-atom gl) gl))))))
+              (reset! async-copy-state-atom (scene-async/finish-image! (scene-async/begin-read! @async-copy-state-atom gl) gl))
+              ;; call frame-selection if it's the very first aabb change for the scene
```

Comment:
This code moved out from graph here after review. It's not perfect, and will be checked more often than in previous implementation, where we catch the moment of AABB change. But I took a look of the code around, and it seems like that's the way of doing it. Let me know if I'm wrong here